fedimint-client/src/client.rs

```diff
@@ -1881,6 +1876,11 @@ impl Client {
         self.log_event_added_transient_tx.subscribe()
     }
 
+    /// Get a receiver that signals when new events are added to the event log
+    pub fn log_event_added_rx(&self) -> watch::Receiver<()> {
+        self.log_event_added_rx.clone()
+    }
+
```

Comment:
Out of curiosity, what is this used for? If you are consuming the event log you should already get notified, right?